OpenRA.Game/Settings.cs

```diff
@@ -336,9 +336,16 @@ public class ChatSettings
 		public bool ConnectAutomatically = false;
 	}
 
+	public class LobbySettings
+	{
+		public string Faction = "Random";
```

Comment:
@pchote If we want to improve our support for parallel installations of OpenRA mods/games, I wonder if we should really make names persistent across mods. Sure, for our shipping mods it would be more convenient because players are used to it, but for 3rd-party mods not so much, and there might actually be players who want to use different names for different mods.